ical.js

```diff
@@ -577,7 +591,9 @@ module.exports = {
               // If the original date has a TZID, add it
               if (curr.start.tz) {
                 const tz = getTimeZone(curr.start.tz);
-                rule += `;DTSTART;TZID=${tz}:${curr.start.toISOString().replace(/[-:]/g, '')}`;
+                const tzoffset = moment.tz(tz).utcOffset() * -60000;
```

Comment:
Why `-60000` is required here? This smells like a modification for DST/non-DST which might not be valid in other time zones. Please explain.

Reply:
In fact, the minus in `-60000` compensates with the minus in `Date(curr.start - tzoffset)` in the next line in order to avoid a plus to be interpreted as implicit string concatenation. The factor `60000` is for conversion from minutes to milliseconds.